compiler/src/iree/compiler/Codegen/LLVMGPU/KernelConfig.cpp

```diff
@@ -520,6 +522,19 @@ setMatmulVectorDistributionConfig(mlir::FunctionOpInterface entryPoint,
   for (int64_t batch : contractionDims->batch) {
     workgroupTileSizes[batch] = 1;
   }
+
+  // Tile all m, n, and k dimensions to 1 except the inner most. Unit dims
+  // from this tiling are folded before vectorization.
+  for (int64_t m : ArrayRef<unsigned int>(contractionDims->m).drop_back()) {
```

Comment:
I think you can do:
```suggestion
  for (int64_t m : llvm::drop_end(contractionDims->m)) {
```
also below.